landing/src/components/modules/filters.tsx

```
@@ -1,18 +1,22 @@
 
-const frameworksList = [
-  { value: "react", label: "React", icon: Turtle },
-  { value: "angular", label: "Angular", icon: Cat },
-  { value: "vue", label: "Vue", icon: Dog },
-  { value: "svelte", label: "Svelte", icon: Rabbit },
-  { value: "ember", label: "Ember", icon: Fish },
-];
+import { ResourcesIds } from "@bibliothecadao/eternum";
+
+const frameworksList = Object.values(ResourcesIds)
+  .filter((value) => isNaN(Number(value)))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)